README.md

```diff
@@ -278,48 +278,49 @@ your operating system:
 
 1. The current working directory (from where you started OpenTTD)
 
-	For non-Windows operating systems OpenTTD will not scan for files in this
-	directory if it is your personal directory, i.e. '~/', or when it is the
-	root directory, i.e. '/'.
+    For non-Windows operating systems OpenTTD will not scan for files in this
+    directory if it is your personal directory, i.e. '~/', or when it is the
+    root directory, i.e. '/'.
+
 2. Your personal directory
     - Windows:
         - `C:\My Documents\OpenTTD` (95, 98, ME)
         - `C:\Documents and Settings\<username>\My Documents\OpenTTD` (2000, XP)
-        - `C:\Users\<username>\Documents\OpenTTD` (Vista, 7)
-    - Mac OSX: `~/Documents/OpenTTD`
+        - `C:\Users\<username>\Documents\OpenTTD` (Vista, 7, 8.1, 10)
+    - Mac OS X: `~/Documents/OpenTTD`
     - Linux: `$XDG_DATA_HOME/openttd` which is usually `~/.local/share/openttd`
      when built with XDG base directory support, otherwise `~/.openttd`
 3. The shared directory
     - Windows:
         - `C:\Documents and Settings\All Users\Shared Documents\OpenTTD` (2000, XP)
-        - `C:\Users\Public\Documents\OpenTTD` (Vista, 7)
-    - Mac OSX: `/Library/Application Support/OpenTTD`
+        - `C:\Users\Public\Documents\OpenTTD` (Vista, 7, 8.1, 10)
+    - Mac OS X: `/Library/Application Support/OpenTTD`
```

Comment:
`macOS`

Reply:
I know that the name was changed from OS X to macOS some time ago, but I have left the old name on purpose, because I don't have any of Apple's devices to test if the path `/Library/Application Support/OpenTTD` is still unchanged in macOS 10.13.

So, please, tell me if I should rename them or not? Or maybe should I do something another with them?

Reply:
I know that the name was changed from OS X to macOS some time ago, but I have left the old name on purpose, because I don't have any of Apple's devices to test if the path `/Library/Application Support/OpenTTD` is still unchanged in recent versions of macOS as well as if OpenTTD will even work on them.

So, please, tell me if I should rename them or not? Or maybe should I do something another with them?

Reply:
I'd say the most important is to stay consistent throughout the file, which is also what you've been going for. Since Apple's official name for the OS is now "macOS", go with that throughout. Whether every detail about file locations etc. still applies to most recent versions is a different concern, although I doubt these things have changed.